renderer/components/editor/controls/right.js

```diff
@@ -7,17 +8,26 @@ import {connect, VideoContainer, EditorContainer} from '../../../containers';
 import formatTime from '../../../utils/format-time';
 
 class RightControls extends React.Component {
+  state = {}
+
+  componentDidMount() {
+    const settings = electron.remote.require('./common/settings');
+    const recordAudio = settings.get('recordAudio');
+    this.setState({recordAudio});
```

Comment:
We shouldn't need the state, recordAudio, or electron at all now. The hasAudio coming from the container should be enough